### DISCUSSION TITLE
Add the thermostat to the list of supported equipment!

### DISCUSSION THREAD

**Discussion by Andy777777777** [Koenkk/zigbee2mqtt#12142] [Devices]
Moes BHT-002-GBLZ in new design!
[https://aliexpress.ru/item/1005002668508718.html?gatewayAdapt=glo2rus&sku_id=12000021629610447&spm=a2g0s.12269583.0.0.11e953e65as1Xk](url)
Moes BHT-3000-GBLZ, defined as BHT-002-GBLZ (( 
Please correct!

**Comment by ovizii** [comment]
I’m not sure if I understand, do you mean to say that yigbee2mqtt does actually support the 3000 series and not the 2000 series? Really curious as I was going to buy a ‎BHT-3000GCLW until I realized it's not supported by zigbee2mqtt.
